### PR TITLE
depend on version 2 of docker-py

### PR DESCRIPTION
**Pull request by TomasTomecek**
which is now available as 'docker' in PyPI and as a `python{2,3}-docker`
in Fedora.

Related: https://github.com/projectatomic/atomic/pull/894

I'm assuming that I shouldn't make the change in `.redhat-ci` file.

### CODE REVIEW

File: .redhat-ci.Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN dnf install -y \
         python-dbus \
         pylint \
         python-slip-dbus \
-        python-docker-py \
+        python2-docker \
```
Comment (baude) [inline .redhat-ci.Dockerfile]: are these rpm changes already propagated to f25, rawhide, centos, and rhel?

Comment (rhatdan) [inline .redhat-ci.Dockerfile]: Not in RHEL/Centos and maybe not in F25.

Comment (baude) [inline .redhat-ci.Dockerfile]: @rhatdan exactly, i think this only applies to rawhide at this point.

Comment (TomasTomecek) [inline .redhat-ci.Dockerfile]: Correct, this is only rawhide now. There is no plan to add the package to F25.

Comment (rhatdan) [inline .redhat-ci.Dockerfile]: @jlebon Any way to handle this?

Comment (jlebon) [inline .redhat-ci.Dockerfile]: But the container is based on F24 (should probably be updated to F25 now), so this isn't going to resolve, right?

Sounds instead like we should set up a rawhide tester (though of course, this issue there is that you'll have to deal with false positives).

Comment (TomasTomecek) [inline .redhat-ci.Dockerfile]: It could also be possible to pull the package directly from koji. As a workaround before having F26.

Comment (jlebon) [inline .redhat-ci.Dockerfile]: Hmm, if we go that way, I think I'd rather make it a new testsuite (they're delimited by `---` in the YAML). It's best if the F25/24 ones reflect as close as possible the actual state we release in.

Comment (TomasTomecek) [inline .redhat-ci.Dockerfile]: Is that something I should do?

Comment (jlebon) [inline .redhat-ci.Dockerfile]: So another issue here is that F25 has Python v3.5, while rawhide has v3.6. Are we expecting incompatibilities here? Given that we'll be branching soon, maybe let's leave the redhat-ci testsuites for now and circle around to it when we add support for F26. I opened https://github.com/jlebon/redhat-ci/issues/28 to track that work.

Comment (rhatdan) [inline .redhat-ci.Dockerfile]: Yes I think we should wait for F26 to make this change.  I don't believe 3.5 vs 3.6 should be an issue.

Comment (TomasTomecek) [inline .redhat-ci.Dockerfile]: Okay, so I'll leave this open and let's revisit once F26 moves forward.

**Comment by TomasTomecek** [comment]
Since Fedora 26 is released, what can I do to complete this?

**Comment by jlebon** [comment]
@TomasTomecek You should be able to rebase and modify `.papr.sh` accordingly.

**Comment by TomasTomecek** [comment]
bot, retest this please

**Comment by TomasTomecek** [comment]
The CI failure:

```
### TIMED OUT AFTER 1800s
```

**Comment by giuseppe** [comment]
I think we can ignore the fedora/26/atomic failure.

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
Looks ready to merge.

**Comment by TomasTomecek** [comment]
\o/

**Comment by ashcrow** [comment]
cc @baude

**Comment by ashcrow** [comment]
@baude / @rhatdan: **bump**

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**